src/framework/filterControl/forms/SearchableMultiSelectFilterForm.spec.jsx

```diff
@@ -0,0 +1,168 @@
+import { TestCaseFactory } from 'react-test-kit';
+import SearchableMultipleSelectFilterForm
+  from './SearchableMultipleSelectFilterForm.jsx';
+import { Filter, FilterOption, OneOfOption } from '../../services';
+
+describe('SearchableMultipleSelectFilterForm', () => {
+  const options = [
+    new OneOfOption('Active', 'Running'),
+    new OneOfOption('Stopped', 'Paused'),
+    new OneOfOption('Archived', 'Archive'),
+  ];
+  const filterOption = new FilterOption({
+    name: 'testFilterOption',
+    comparisonParameters: {
+      oneOfOptions: options,
+    },
+  });
+
+  describe('initial state', () => {
+    it('has options unselected', () => {
+      const props = {
+        filterOption,
+        onAddFilter: () => undefined,
+      };
+
+      const testCase =
```

Comment:
also why new line here and below?

Reply:
the eslint config has a strict 80 char limit and this line goes over that